.github/workflows/test.yml

```diff
@@ -19,12 +19,19 @@ jobs:
           - 3.6
           - 3.8
         odoo-version:
-          - 10.0
           - 11.0
           - 12.0
           - 13.0
           - 14.0
           - 15.0
+        include:
+          - python-version: 2.7
+            odoo-version: 10.0
+        exclude:
+          - python-version: 3.8
+            odoo-version: 11.0
+          - python-version: 3.8
+            odoo-version: 12.0
```

Comment:
Suggestion: Instead of letting the matrix auto-generate permutations of `python-version` and `odoo-version` and having to deal with incompatibilities, maybe we should hard-code the permutations in `include` to match the permutations in  `src/.github/workflows/test.yml`.

Reply:
This is the test of the *template*, why does it depend on the Odoo/python version combination ?